bom/runtime/pom.xml

```diff
@@ -28,6 +28,7 @@
         <jboss-servlet-api_4.0_spec.version>1.0.0.Final</jboss-servlet-api_4.0_spec.version>
         <microprofile-config-api.version>1.3</microprofile-config-api.version>
         <microprofile-context-propagation.version>1.0.1</microprofile-context-propagation.version>
+        <microprofile-metrics.version>2.1.0</microprofile-metrics.version>
```

Comment:
I think we probably want 2.2.0?

Reply:
Just saw 2.2 PR was open after this one :)
Will move to 2.2

Reply:
I find it quite disturbing that we need to declare the same version in two different `pom.xml` files, we might probably want to introduce a property to avoid the danger of having two different versions. But that's not the point of this PR.

Reply:
@jmartisk now that we switch to the latests smallrye-fault-tolerance and smallrye-metrics I can remove this as both use the 2.2 version

Reply:
done